client/blocks/login/index.jsx

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+//import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import FormTextInput from 'components/forms/form-text-input';
+import { localize } from 'i18n-calypso';
+import { loginUser } from 'state/login/actions';
+
+//const Login = ( { title, buttonText, legalText, translate, children } ) => {
+class Login extends Component {
+	static defaultProps = {
+		title: '',
+		buttonText: '',
+		legalText: ''
+	};
+
+	state = {
+		usernameOrEmail: '',
+		password: '',
+	};
+
+	onChangeField = ( event ) => {
+		this.setState( {
+			[ event.target.name ]: event.target.value
+		} );
+	};
+
+	onSubmitForm = ( event ) => {
+		event.preventDefault();
+		this.props.loginUser( this.state.usernameOrEmail, this.state.password );
+	};
+
+	render() {
+		const buttonText = this.props.buttonText || this.props.translate( 'Sign in' );
+		return (
+			<div className="login">
+				<form onSubmit={ this.onSubmitForm }>
+					<Card className="login__form-userdata">
+						<div className="login__form-header">
+							<div className="login__form-header-title">{ this.props.title }</div>
+						</div>
+						{ this.props.children }
+						<div className="login__form-userdata">
+							<label className="login__form-userdata-username">
+								{ this.props.translate( 'Username or email' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Usernames or Emails' )` *ES Score: 13.85*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/login-component)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).